src/components/cloneGoals.js

```diff
@@ -0,0 +1,40 @@
+import { createGoal } from '../goals/actions'
+import { selectGoal } from '../selection/actions'
+import { addMemberOfGoal } from '../goal-members/actions'
+import moment from 'moment'
+
+export default function cloneGoals(store, goalsToClone, goalMembers, goals) {
```

Comment:
Does `goals` get used anywhere? remove if not

Reply:
.dispatch(
        createGoal.create({
          goal: {
            ...goals[value],
            timestamp_created: moment().unix(),
          },
          maybe_parent_address: null,
        })
      )

Reply:
oh right, I see 👍